### ISSUE TITLE
Remove changing boot partition

### ISSUE THREAD

**Issue by urpylka** [urpylka/img-tool#3]
```bash
stamp "Change boot partition"
sed -i 's/root=[^ ]*/root=\/dev\/mmcblk0p2/' /boot/cmdline.txt
sed -i 's/.*  \/boot           vfat    defaults          0       2$/\/dev\/mmcblk0p1  \/boot           vfat    defaults          0       2/' /etc/fstab
sed -i 's/.*  \/               ext4    defaults,noatime  0       1$/\/dev\/mmcblk0p2  \/               ext4    defaults,noatime  0       1/' /etc/fstab
```

**Comment by urpylka** [comment]
Now doesn't work load by usb-drive

**Comment by urpylka** [comment]
The problem was solved in 1200b27ee3c1ed5202267e2565b0a460ea2fa08b and 15005898aa8b89717990f4ae9399c90490e3ea36.
